### PR TITLE
Redid shader options to be more intuitive, pass 1

### PR DESCRIPTION
**Pull request by WoMspace**
please let me know if I missed any settings xd
I will keep working at it but it's 5:30am sooooooooooooo :)

### CODE REVIEW

File: shaders/final.vsh [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+#define FinalR 1.0 //[0 0.1 0.2 0.3 0.4 0.5 0.6 0.7 0.8 0.9 1.0 1.1 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 2 2.1 2.2 2.3 2.4 2.5 2.6 2.7 2.8 2.9 3 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8 3.9 4 4.1 4.2 4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9 6 6.1 6.2 6.3 6.4 6.5 6.6 6.7 6.8 6.9 7 7.1 7.2 7.3 7.4 7.5 7.6 7.7 7.8 7.9 8 8.1 8.2 8.3 8.4 8.5 8.6 8.7 8.8 8.9 9 9.1 9.2 9.3 9.4 9.5 9.6 9.7 9.8 9.9 10.0]
```
Comment (JoubaMety) [inline shaders/final.vsh:4]: coulda use some moar granular control, ngl :kek:

File: shaders/shaders.properties [hunk 2]
```diff
@@ -73,38 +73,111 @@ alphaTest.gbuffers_skytextured=false
...
+[Lighting] \
+[Terrain] \
+[Water] \
+[Camera] \
+[PostProcessing] \
+[Debug]
+
+# [Atmosphere] -----------
+screen.Atmosphere.columns=1
+screen.Atmosphere=[Clouds] [Fog] [Sky]
```
Comment (JoubaMety) [inline shaders/shaders.properties:91]: I see missed opportunity to format it similarly to like how you did with lines 79-87

File: shaders/shaders.properties [hunk 3]
```diff
@@ -73,38 +73,111 @@ alphaTest.gbuffers_skytextured=false
...
+screen.Purkinje_effect = Purkinje_strength Purkinje_Multiplier Purkinje_R Purkinje_G Purkinje_B
+# [Tonemapping]
+screen.Tonemapping.columns=1
+screen.Tonemapping = TONEMAP USE_ACES_COLORSPACE_APPROXIMATION SATURATION CROSSTALK FinalR FinalG FinalB
+
+# [Debug] -----------
 screen.Debug = <empty> DEBUG DEBUG_PROGRAM <empty> DEBUG_BRIGHTNESS DRAW_DEBUG_VALUE <empty>  [misc]
+screen.misc = *
 
+########## END OF WOMSPACE MENU ###########
```
Comment (JoubaMety) [inline shaders/shaders.properties:180]: self-inserted advertisement, kinky :lenny:

File: shaders/shaders.properties (hunk 2 above)
Comment (WoMspace) [reply shaders/shaders.properties:91]: Intentional.

File: shaders/final.vsh (hunk 1 above)
Comment (WoMspace) [reply shaders/final.vsh:4]: You are joking... Right?

**Pull request merged**